### PR TITLE
Fix multi upload hash issue

### PR DESCRIPTION
**Pull request by cloakd**

### CODE REVIEW

File: src/methods/upload-multiple-files.ts [hunk 1]
```diff
@@ -150,11 +150,11 @@ export default async function uploadMultipleFiles(
     );
     let msgSig;
     if (!this.wallet.signMessage) {
-      msgSig = nacl.sign.detached(msg, this.wallet.payer.secretKey);
+      msgSig = nacl.sign.detached(Buffer.from(msg, "utf-8"), this.wallet.payer.secretKey);
```
Comment (WillSnowdon) [inline src/methods/upload-multiple-files.ts:153]: second arg not needed with `msg` being of type `Uint8Array` here?

Comment (cloakd) [reply src/methods/upload-multiple-files.ts:153]: idk, you can test if it works without - just going off official sol docs.

File: src/methods/upload-multiple-files.ts [hunk 2]
```diff
@@ -150,13 +150,13 @@ export default async function uploadMultipleFiles(
     );
     let msgSig;
     if (!this.wallet.signMessage) {
-      msgSig = nacl.sign.detached(msg, this.wallet.payer.secretKey);
+      msgSig = nacl.sign.detached(Buffer.from(msg), this.wallet.payer.secretKey);
     } else {
-      msgSig = await this.wallet.signMessage(msg);
+      msgSig = await this.wallet.signMessage(Buffer.from(msg));
```
Comment (WillSnowdon) [inline src/methods/upload-multiple-files.ts:155]: just had a chance to have a quick look at Phantom docs and they say Buffer is not needed, you can provide the result of `encodeText` like before but with a second arg of "utf-8"

https://docs.phantom.app/integrating/extension-and-in-app-browser-web-apps/signing-a-message

File: src/methods/upload-multiple-files.ts (hunk 1 above)
Comment (khaIilnafis) [reply src/methods/upload-multiple-files.ts]: resolved this issue in https://github.com/GenesysGo/shadow-drive/pull/35/commits/da43228fa649c4a2ef4846b2ab1b4fffa49354db

**Pull request merged**